### PR TITLE
FLAP-153 check the ability to remove nodes

### PR DESCRIPTION
**Pull request by amneet270**
# Description

This test creates new a new node and then deletes it

## Link to work item

test for the user story https://flapflap.atlassian.net/browse/FLAP-80

### CODE REVIEW

File: tests/test_bugs.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from selenium.webdriver.common.by import By
```
Comment (theNafsters) [inline tests/test_bugs.py]: Is this test necessary? We want to test our features and not the bug that is present

Comment (amneet270) [reply tests/test_bugs.py]: I don't think having tests for the bugs will be all bad, this way we can make sure the bug was fixed properly

Comment (theNafsters) [reply tests/test_bugs.py]: Okay, thats fine, we can leave it then . Ill approve the PR since the tests were written properly

**Review by theNafsters — APPROVED** [review]
Good job amneet!!

**Pull request closed without merging**